pkg/controller/middleware/membership.go

```diff
@@ -83,6 +83,7 @@ func LoadCurrentMembership(cacher cache.Cacher, db *database.Database, h render.
 
 			// Save the membership and realm on the context.
 			ctx = controller.WithMembership(ctx, membership)
+			ctx = controller.WithRealm(ctx, membership.Realm)
```

Comment:
I intentionally didn't set this because I don't want to pull the realm from the context. It should always be pulled from the membershpi.

Reply:
I don't think there is a membership for API based calls.
https://github.com/google/exposure-notifications-verification-server/blob/8a910cfa82ee86fa54762a8228d339bbb44876e4/pkg/controller/middleware/apikey.go#L110

If we want to remove WithRealm and always take either membership or authapp and re-look it up, we could. If so I'd rather also remove controller.WithRealm entirely